cmd/kola/kola.go

```diff
@@ -140,6 +140,12 @@ func writeProps() error {
 		InstallerImageBaseURL string `json:"installer"`
 		ImageURL              string `json:"image"`
 	}
+	type ESX struct {
+		Server     string `json:"esx_server"`
+		User       string `json:"esx_user"`
+		Password   string `json:"esx_password"`
+		BaseVMName string `json:"esx_base_vm_name"`
+	}
```

Comment:
They serve completely different purposes, but happen to have the same contents at the moment.  The current approach is the right one.